data/alpha/field/government-organisation.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: government-organisation
+phase: alpha
+register: ''
```

Comment:
`s/register: ''/register: government-organisation/`

As this is the primary key field, we put the register name on this line. :)